docs/_config.yml

```diff
@@ -45,7 +45,8 @@ sphinx:
     html_theme_options: {}
     pygments_style: solarized-dark
     suppress_warnings: ["myst.header"]
-    exclude_patterns: ['**/_*.md']  # includes
+    # build the generated files in this repo to preview them
```

Comment:
Should we just delete both lines here?

Reply:
I commented in case we want to go back to the old way or do something different